src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.spec.ts

```diff
@@ -244,7 +248,9 @@ describe('CustomerTableListComponent', () => {
       _status: false,
       btnColor: 'btn-danger',
       btnIcon: 'fa-arrow-circle-down',
+      btnIconTwo:  'fa-caret-check',
```

Comment:
use only a space after `:`, replicate this change in the other parts with `btnIconTwo: 'fa-caret-check'`

Reply:
done